packages/dds/tree/src/simple-tree/jsonSchema.ts

```diff
@@ -0,0 +1,208 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { SimpleNodeSchemaKind } from "./simpleSchema.js";
+
+/**
+ * The fully-qualified {@link TreeNodeSchemaCore.identifier}.
+ * @example Schema `com.myapp.foo` would be referenced via `#/$defs/com.myapp.foo`.
```

Comment:
It's not immediately clear to me if the `string: JsonSchemaId` itself is "com.myapp.foo" or "#/$defs/com.myapp.foo". 
Saying "fully qualified" could mean either, sort of. It's obvious that it's the shorter one when considered alongside `JsonRefPath` below, but it's unclear just from this doc alone - maybe it should link to `JsonRefPath`.

Reply:
Oof, this example should have been removed. It belongs on `JsonRefPath`. I'll remove this. This is strictly the identifier.

Might be worth renaming to just "SchemaIdentifier" to be less JSON-specific (it really isn't).

Reply:
Or re-using an existing type alias elsewhere, if we have one.

Reply:
```suggestion
```